GearBot/Cogs/Moderation.py

```diff
@@ -85,6 +89,54 @@ def _can_act(action, ctx, user: discord.Member, check_bot=True):
         else:
             return False, Translator.translate(f'{action}_not_allowed', ctx.guild.id, user=user)
 
+    @commands.group()
+    @commands.guild_only()
+    async def emote(self, ctx):
+        if ctx.subcommand_passed is None:
+            return await ctx.send("Please pass a subcommand")
```

Comment:
can you execute help instead?

also there is no need to return the message you send, it can be useful to do sometimes to exit out early, but other then that there isn't any point in doing it
```suggestion
            await ctx.invoke(self.bot.get_command("help"), "emote")
```